src/nvim/testdir/test_regexp_latin.vim

```diff
@@ -0,0 +1,39 @@
+" Tests for regexp in latin1 encoding
```

Comment:
beyond possibly testing `:scriptencoding` command, this doesn't seem to test anything that the utf8 test already tests? If the purpose was to test `enc=latin1` I'll suggest just to remove this test, like we did for the latin1 spell test.